imap_processing/tests/ultra/unit/test_decom_apid_883.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from imap_processing.ultra.l0.decom_ultra import decom_ultra_apids
+from imap_processing.ultra.l0.ultra_utils import UltraParams
+
+
+@pytest.fixture()
+def decom_ultra(ccsds_path_tof, xtce_path):
+    """Data for decom_ultra"""
+    data_packet_list = decom_ultra_apids(
+        ccsds_path_tof, xtce_path, UltraParams.ULTRA_TOF.value.apid[0]
+    )
+    return data_packet_list
+
+
+def test_image_ena_phxtof_hi_ang_decom(decom_ultra, tof_test_path):
+    """This function reads validation data and checks that decom data
+    matches validation data for image rate packet"""
+
+    df = pd.read_csv(tof_test_path, index_col="SequenceCount")
+
+    assert (df.Spin == decom_ultra["SPIN"]).all()
```

Comment:
I'd suggest numpy's testing assertions for arrays of values.
```suggestion
    np.testing.assert_array_equal(df.Spin, decom_ultra["SPIN"])
```